css/style/toast.css

```diff
@@ -1,41 +1,65 @@
 #toast {
-    visibility: hidden;
-    min-width: 15.625rem;
-    margin-left: -7.81rem;
-    background-color: var(--text-color);
-    color: var(--text-light);
-    text-align: center;
-    border-radius: 2px;
-    padding: 16px;
-    position: fixed;
-    z-index: 6;
-    left: 50%;
+  visibility: hidden;
+  min-width: 15.625rem;
```

Comment:
you can just use 15.6 rem instead of .625

Reply:
Yeah, actually I converted px into rem and wrote exact value